arch/platform/openmote/dev/antenna.c

```diff
@@ -73,6 +79,20 @@ antenna_internal(void)
   GPIO_WRITE_PIN(ANTENNA_BSP_RADIO_BASE, ANTENNA_BSP_RADIO_INT, 1);
 }
 /*---------------------------------------------------------------------------*/
+void
+antenna_select_atmel(void)
```

Comment:
What is the difference between these two functions?

Reply:
Both OpenMote-CC2538 and OpenMote-B boards contain an antenna switch.

- On OpenMote-CC2538, the switch is used to connect the output of the radio to either the internal (chip antenna) or external antenna (SMA connector).

- For OpenMote-B, on the other hand, the switch selects which output of the radio goes to the SMA connector - either Atmel's AT86RF215 radio or TI CC2538 radio.

On both boards, the RF switch is connected to the same GPIO pins - I have created new functions (which basically do the same thing as the original ones) just to avoid any misunderstandings about function names.

At this point I am not sure if I should create a separate file for each board, or if we should use the originally defined functions and just add some documentation describing the differences between the boards?

Reply:
Initially I thought they set/clear the same pins, but they set/clear the opposite pins, so ignore the comment.

Reply:
I will update (rebase) this PR when the #2684 is accepted